core/tests/webdriverio_utils/TopicEditorPage.js

```diff
@@ -173,19 +258,31 @@ var TopicEditorPage = function() {
       'Create subtopic modal taking too long to disappear.');
   };
 
-  this.navigateToTopicEditorTab = async function() {
-    await action.click('Topic Editor Tab', topicEditorTab);
+  this.addConceptCardToSubtopicExplanation = async function(skillName) {
+    await action.click('RTE input', subtopicPageContentButton);
+    // Chaining selectors is selecting the non-interceptable element.
+    // eslint-disable-next-line oppia/e2e-practices
+    var conceptCardButton = $('#cke_124');
```

Comment:
What is the significance of 124? Is it the same on every run? Where do we specify cke_124 in the codebase?

Reply:
Well, you can't just leave it like this without explanation then -- I think this needs a long comment in the code (with more detail than what you've just written) to explain why we're doing this and why alternatives are not possible. Otherwise other devs will have no idea what is going on.

Reply:
Done.